src/Objs/Product/ProductComponent.tsx

```diff
@@ -23,7 +23,7 @@ provideComponent(Product, ({ page }) => {
     .map(toPlainParameter)
 
   return (
-    <>
+    <main id="main">
```

Comment:
This seems to break the design, at least on https://main-tag-refactoring.scrivito-portal-app.pages.dev/de/produkte-ed043d1606eaf539 . Any ideas?

Reply:
The problem appears to be flex layout. Before, every `section` inside `#root` was a flex child, now the whole `#main` wraps several sections, which leads to different flex heights. We must keep `flex` to keep the footer at the bottom.
How about setting `section { flex-shrink: 0; }`?

Reply:
After a discussion with Alex G. there are two options:
1. We move the top sections which are no longer part of `#main` to the `header`. Note: This would be the best solution from a semantic point of view.
2. If this is not easily possible, we'd remove the "#root 100% height" rules. This moves the footer from the bottom of the window to the bottom of the content, but we don't consider the footer position critical.

Reply:
"We move the top sections which are no longer part of #main to the header. " is my goal - once I'm done will all refactorings.

Reply:
I'll disable "#root 100% height" for now.